docs/source/output_sanitize.cfg

```diff
@@ -0,0 +1,73 @@
+[regex1]
```

Comment:
Please do not commit this ` docs/source/output_sanitize.cfg` file.  To prevent this file from coming back in future commits, add it to your `.gitignore`.

Reply:
Still have to delete this `docs/source/output_sanitize.cfg` file.